design_documents/accessing_emitted_events.md

```diff
@@ -1,4 +1,4 @@
-# Accessing emitted events
+# Accessing Emitted Events
 
 <!-- TOC -->
 * [Accessing emitted events](#accessing-emitted-events)
```

Comment:
If we are changing the contents of this doc, capitalization of the table of contents should be updated as well.